plugins/kinetic-dp/fu-kinetic-dp-connection.c

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright (C) 2015 Richard Hughes <richard@hughsie.com>
+ * Copyright (C) 2016 Mario Limonciello <mario.limonciello@dell.com>
+ * Copyright (C) 2017 Peichen Huang <peichenhuang@tw.synaptics.com>
+ * Copyright (C) 2021 Jeffrey Lin <jlin@kinet-ic.com>
+ * Copyright (C) 2022 Hai Su <hsu@kinet-ic.com>
+ *
+ * SPDX-License-Identifier: LGPL-2.1+
+ */
+
+#include "config.h"
+
+#include "fu-kinetic-dp-common.h"
+#include "fu-kinetic-dp-connection.h"
+
+#define UNIT_SIZE     32
+#define MAX_WAIT_TIME 3 /* unit : second */
+
+struct _FuKineticDpConnection {
+	GObject parent_instance;
+	gint fd; /* not owned by the connection */
+};
+
+G_DEFINE_TYPE(FuKineticDpConnection, fu_kinetic_dp_connection, G_TYPE_OBJECT)
+
+static void
+fu_kinetic_dp_connection_init(FuKineticDpConnection *self)
+{
+	g_debug("connection instance initialized.");
```

Comment:
i don't think we want/need all these g_debugs like this

Reply:
removed